config/locales/en.yml

```diff
@@ -223,13 +223,14 @@ en:
     copyright: "2020 Houdini Project"
   login:
     header: "Login"
-    message: "Welcome Back"
+    message: "Please Enter Login Information"
```

Comment:
I'm wondering if this is the best name for this this particular message. If someone sees `login.message` in the codebase, would they know or have a hunch that this is the message that goes there?

Reply:
They probably won't. I will fix this.